### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by queencre**
## Description

This PR simply updates `README.md` file.

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Appropriate labels applied
- [ ] Targeted PR against correct branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://go.dev/blog/godoc).
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Comment by queencre** [comment]
I thought of adding `Resources` section  in `README.md` file to specify block explorer(s), wallet(s), and any resources that are informative. However, I think it is okay as is for now and we can add it with a separate PR when the resources are determined after Mainnet.

File: README.md [hunk 1]
```diff
@@ -1,55 +1,105 @@
...
+```bash
+crescentd version --long
+```
+
+If `crescentd` is installed correctly, the following information is returned:
+
+```
+name: crescent
+server_name: crescentd
+version: TBD
```
[the remark this answers is not in the document]
Comment (queencre) [reply README.md]: I think you are referring to skipping lines from  67, right? (the return output part)

**Review by crypin — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,55 +1,94 @@
...
-If you haven't already, install Golang by following the [official docs](https://golang.org/doc/install). Make sure that your `GOPATH` and `GOBIN` environment variables are properly set up.
 
-This project uses customized cosmos-sdk, Please check the difference on [here](https://github.com/crescent-network/cosmos-sdk/compare/v0.44.5...v1.0.2-sdk-0.44.5).
+## What is Crescent?
 
-| Requirement           | Notes             |
-|-----------------------|-------------------|
-| Go version            | Go1.16 or higher  |
-| Cosmos SDK (custom)   | v1.0.2-sdk-0.44.5 |
+Crescent is a DeFi Hub that provides innovative and powerful tools that empower users’ digital assets for maximizing their financial returns while manage associated risks in the most efficient way. In the base layer, Crescent core has a DEX functionality equipped with several unique characteristics.
```
Comment (kingcre) [inline README.md:26]:
```suggestion
Crescent is a DeFi Hub that provides innovative and powerful tools that empower users’ digital assets for maximizing their financial returns while managing associated risks in the most efficient way. In the base layer, Crescent core has a DEX functionality equipped with several unique characteristics.
```

File: README.md [hunk 3]
```diff
@@ -1,55 +1,94 @@
...
+
+**Step 1. Install Golang**
+
+Go version [1.16](https://go.dev/doc/go1.16) or higher is required for Crescent Core.
+
+If you haven't already, install Go by following the [official docs](https://golang.org/doc/install). Make sure that your `GOPATH` and `GOBIN` environment variables are properly set up.
+
+**Step 2. Get Crescent Core source code**
+
+Use `git` to retrieve Crescent Core from the [official repo](https://github.com/crescent-network/crescent/) and checkout the `main` branch. This branch contains the latest stable release, which will install the `crescentd` binary.
```
Comment (kingcre) [inline README.md:51]: I don't think `main` branch contains a **stable** release.

**Review by kingcre — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (queencre) [reply README.md:51]: Let me change that to just `the latest release`.

**Review by kingcre — APPROVED** [review]
LGTM

**Pull request merged**